locales/mr.js

```
@@ -38,39 +49,46 @@ const mr = {
 
     // Proxy
     "useproxytitletext": "प्रॉक्सी बायपास",
-    "useproxyText": "जर शोध सूचना कार्यरत नसतील",
+    "useproxyText": "जर शोध सूचना चालत नसेल",
     "ProxyText": "CORS बायपास प्रॉक्सी",
     "ProxySubtext": "आपला स्वतःचा CORS बायपास प्रॉक्सी जोडा",
     "HostproxyButton": "आपला प्रॉक्सी होस्ट करा",
 
     // Location
-    "UserLocText": "आपले स्थान प्रविष्ट करा",
+    "useGPS": "GPSचा उपयोग",
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)